code/game/objects/structures/watercloset.dm

```diff
@@ -440,8 +443,8 @@
 			else
 				B.deductcharge(1)
 			user.visible_message( \
-				"[user] was stunned by his wet [O].", \
-				"<span class='warning'>You have wet \the [O], it shocks you!</span>")
+				"<span class='warning'>[user] was stunned by \his wet [O.name]!</span>", \
```

Comment:
Standard form is to display the name anyway in this case, so it doesn't really matter